css/main.scss

```diff
@@ -1,4 +1,4 @@
-@import url(//fonts.googleapis.com/css?family=Roboto:100,300,400);
+@import url(http://fonts.googleapis.com/css?family=Roboto:100,300,400);
```

Comment:
this isnt always good as it results in mixed content warnings with https connections